SandsTrilogyKiller.csproj

```diff
@@ -31,6 +31,9 @@
     <ErrorReport>prompt</ErrorReport>
     <WarningLevel>4</WarningLevel>
   </PropertyGroup>
+  <PropertyGroup>
+    <ApplicationManifest>app.manifest</ApplicationManifest>

```

Comment:
Is this needed now because admin rights are required to set affinity and priority? Or was it actually already necessary with the previous feature set?

Reply:
It is necessary to set the real-time priority, anything below that its not needed. But in general I thought it would be a good idea as well because there were often complaints of it not working where people would usually launch game as admin but not the program.

Reply:
I believe it would be better to just use the high priority, or rather, I don't see the reason why we should use real-time priority.